00-HTML-CSS-basics/sign-up-form.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Sign Up Form</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <form>
+      First name: <input type="text" name="first-name" id="first-name" value="" required="" />
```

Comment:
It is not mandatory but is a good practice (also for accessibility) to have a label for each input.